solr/contrib/ltr/src/java/org/apache/solr/ltr/LTRRescorer.java

```diff
@@ -53,6 +53,21 @@ public LTRRescorer(LTRScoringQuery scoringQuery) {
     this.scoringQuery = scoringQuery;
   }
 
+  final protected static Comparator<ScoreDoc> docComparator = (a, b) -> a.doc - b.doc;
```

Comment:
```suggestion
  final private static Comparator<ScoreDoc> docComparator = (a, b) -> a.doc - b.doc;
```

i.e. minimum necessary visibility.

And I've seen `Comparator.comparingInt` used elsewhere in the code base recently and would be curious if using that here too might be an alternative, what do you think?

Reply:
Oh yes, I think that it is a absolutely valid alternative! I'll apply this suggestion